tests/vmi_multus_test.go

```diff
@@ -158,8 +158,12 @@ var _ = Describe("[Serial]Multus", func() {
 	})
 
 	Describe("[rfe_id:694][crit:medium][vendor:cnv-qe@redhat.com][level:component]VirtualMachineInstance using different types of interfaces.", func() {
-		Context("VirtualMachineInstance with cni ptp plugin interface", func() {
+		const (
+			CIDR1 = "10.1.1.1/24"
+			CIDR2 = "10.1.1.2/24"
+		)
```

Comment:
You defined it here by not used it in all places below.
If you want to use it only at specific places, define it closer to the usage please.